.github/workflows/spec-update.yaml

```diff
@@ -16,7 +16,7 @@ on:
       file-ref:
         description: "Branch or tag to checkout the spec file from"
         required: false
-        default: main
+        default: v1-main
```

Comment:
[req] `file-ref` is the branch name of the other service, where spec is defined, not our repo. It is default to `main`.